### ISSUE TITLE
Fix `function-linear-gradient-no-nonstandard-direction` test cases

### ISSUE THREAD

**Issue by romainmenke** [stylelint/stylelint#6986]
I think a typo happend in the test cases.

https://github.com/stylelint/stylelint/blob/fb302e244de0309898bebd1f2ed4565fa774c5b7/lib/rules/function-linear-gradient-no-nonstandard-direction/__tests__/index.js#L10-L39

```js
{
	code: '.foo { background: linear-gradient(to top, #fff, #000; )}',
},
```

```css
.foo {
	background: linear-gradient(to top, #fff, #000; )
}
```

I think this was intended to be :

```css
.foo {
	background: linear-gradient(to top, #fff, #000);
}
```

Notice the semicolon position.

------

Should we fix these tests?

**Comment by ybiquitous** [comment]
@romainmenke Good catch! Yes, please open a pull request!
